CodeQL model editor: support saving single/selected models (#3156)
github/vscode-codeql: extensions/ql-vscode/CHANGELOG.md

## [UNRELEASED]

> - In the CodeQL model editor, you can now click individual method rows to save them, instead of having to save the entire library model. [#3156](https://github.com/github/vscode-codeql/pull/3156)

Reviewer: ```suggestion
- In the CodeQL model editor, you can now select individual method rows and save changes to only the selected rows, instead of having to save the entire library model. [#3156](https://github.com/github/vscode-codeql/pull/3156)
```
The current wording makes it sound like it's the act of clicking itself that saves the method row, rather than selecting the row so you can save all selected rows.

Feel free to not take my suggestion though. Any wording is fine but I just wanted to point out the potential confusion.
